src/coreComponents/common/format/table/unitTests/testTable.cpp

```diff
@@ -289,41 +248,85 @@ TEST( testTable, layoutTable )
              );
 }
 
-TEST( testTable, tableSetMargin )
+TEST( testTable, subColumns )
+{
+  {
+    TableLayout tableLayout( {" ",
+                              "Column1",
+                              TableLayout::Column{"Nodes ", TableLayout::Alignment::right, true, {"Locales", "Ghost", "Active"}},
+                              "Column3",
+                              TableLayout::Column{"Column4 ", TableLayout::Alignment::right, true, {"Locales", "Ghost"}},
+                              "Column5"} );
+
+    TableData tableData;
+    tableData.addRow( "min", "125", "375,0001", " YES", 2354654, 562, 43.0, 43.0, 562, 5 );
+    tableData.addRow( "max", "360", "390,1", " YES", 383213213, 712, 48.0, 47.0, 72, 2 );
+
+    TableTextFormatter tableText( tableLayout );
+
+    EXPECT_EQ( tableText.toString( tableData ),
+               "\n--------------------------------------------------------------------------------------------------------\n"
+               "|       |  Column1  |                            Nodes   |  Column3  |           Column4   |  Column5  |\n"
```

Comment:
ofc

Reply:
I messed up what the formatting would do with those setting because of the `TableLayout::Alignment` targetting the headers and not the content. As I said in another comment, please update the value alignement in priority.